rkt/run.go

```diff
@@ -78,6 +78,27 @@ image arguments with a lone "---" to resume argument parsing.`,
 	flagHostsEntries flagStringList
 )
 
+func addIsolatorFlags(cmd *cobra.Command) {
+	cmd.Flags().Var((*appMemoryLimit)(&rktApps), "memory", "memory limit for the preceding image (example: '--memory=16Mi', '--memory=50M', '--memory=1G')")
+	cmd.Flags().Var((*appCPULimit)(&rktApps), "cpu", "cpu limit for the preceding image (example: '--cpu=500m')")
+	cmd.Flags().Var((*appCapsRetain)(&rktApps), "caps-retain", "capability to retain (example: '--caps-retain=CAP_SYS_ADMIN')")
+	cmd.Flags().Var((*appCapsRemove)(&rktApps), "caps-remove", "capability to remove (example: '--caps-remove=CAP_MKNOD')")
+	cmd.Flags().Var((*appSeccompFilter)(&rktApps), "seccomp", "seccomp filter override (example: '--seccomp mode=retain,errno=EPERM,chmod,chown')")
+
+	// For backwards compatibility
```

Comment:
I think you could hide those behind some boolean parameter passed to this function. The backwards compatible deprecated flags won't be necessary for new commands (like `rkt app add` for CRI).